rj_geometry/include/rj_geometry/shape.hpp

```diff
@@ -10,6 +10,9 @@ class Segment;
 /**
  * The shape class provides the interface to all shapes that are subclasses.
  *
+ * TODO(Kevin): do we use boost python anymore? more likely this docstring
+ * was never updated (4/29/21 at time of reading)
```

Comment:
```suggestion
 * was never updated (4/29/22 at time of reading)
```

I forgot the year...